### ISSUE TITLE
stop working

### ISSUE THREAD

**Issue by TAiS46** [WooAf/PoGoBoT#19]
```
[11:31:05] Pokemons in Bag: 161/250.
[11:31:05] Inventory and Pokemon Space, Ready.
[11:31:06] Loot -> Gems: 0, Eggs:  Items: 
[11:31:06] Gained 0 XP.
[11:31:06] ------------> InvalidReponseException
[11:31:06] <------------ Recovering
[11:31:06] Adjusting the Settings.
```

**Comment by dringex** [comment]
[12:01:49] Trying to Login with PTC Credentials...
[12:01:50] Connected! Server is Ready.
[12:01:50] Attempting to Retrieve Inventory and Player Profile...
[12:01:50] Ready to Work.
[12:01:51] Loot -> Gems: 0, Eggs:  Items: 
[12:01:51] Gained 0 XP.
[12:01:51] ------------> InvalidReponseException
[12:01:51] <------------ Recovering
[12:01:51] Adjusting the Settings.
[12:01:51] Trying to Login with PTC Credentials...
[12:01:52] Connected! Server is Ready.
[12:01:52] Attempting to Retrieve Inventory and Player Profile...
[12:01:52] Ready to Work.
[12:01:53] Loot -> Gems: 0, Eggs:  Items: 3 x ItemPokeBall
[12:01:53] Gained 50 XP.
[12:01:53] ------------> InvalidReponseException
[12:01:53] <------------ Recovering

**Comment by WooAf** [comment]
Should be fixed on V4.0.0
